### PR TITLE
OSDOCS-8385: Migrate "Getting Started with Rosa - Obtaining Support" from rosaworkshop.io

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-8385](https://issues.redhat.com//browse/OSDOCS-8385): Migrate "Getting Started with Rosa - Obtaining Support" from rosaworkshop.io

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8385

Link to docs preview:
https://70455--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-support

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can I please have a quick QE review of this PR? Thanks so much!

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-support.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-17
+
+Finding the right help when you need it is important. These are some of the resources at your disposal when you need assistance.
+
+== Adding support contacts
+You can add additional email addresses for communications about your cluster. 
+
+. On the Red Hat OpenShift Cluster Manager user interface (UI), click *select cluster*.
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-support.adoc:17]: Suggestion: 

s/Red Hat OpenShift Cluster Manager/{cluster-manager-first}

File: cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-support.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+. Click the *Support* tab.
+. Click *Add notification contact*, and enter the additional email addresses.
+
+== Contacting Red Hat for support using the UI
+
+. On the OpenShift Cluster Manager UI, click the *Support* tab.
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-support.adoc:23]: Suggestion: 

s/OpenShift Cluster Manager/{cluster-manager}

**Comment by bmcelvee** [comment]
Just two small suggestions. Otherwise, LGTM!

**Pull request merged**